includes/emails/class-woothemes-sensei-email-learner-completed-course.php

```diff
@@ -61,11 +67,10 @@ function trigger( $user_id = 0, $course_id = 0 ) {
 			'passed'			=> $passed,
 		), $this->template );
 
-		// Set recipient (learner)
-		$this->recipient = stripslashes( $this->user->user_email );
-
 		// Send mail
 		$woothemes_sensei->emails->send( $this->recipient, $this->subject, $woothemes_sensei->emails->get_content( $this->template ) );
```

Comment:
which one? 'sensei_before_mail' have to stay. Those two apply filters: it's not me who added this, I think. Maybe it is some legacy resulting from difference between git versions